sources/platform/actors/development/actor_definition/docker.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Dockerfile
+description: Learn about the available Docker images you can use as a base for your Apify Actors. Choose the right base image based on your Actor's requirements and the programming language you're using.
+slug: /actors/development/actor-definition/dockerfile
+sidebar_position: 4
+---
+
+**Learn about the available Docker images you can use as a base for your Apify Actors. Choose the right base image based on your Actor's requirements and the programming language you're using.**
+
+---
+
+When developing an [Actor](/sources/platform/actors/index.mdx) on the Apify platform, you can choose from a variety of pre-built Docker iamges to serve as the base for your Actor. These base images come with pre-installed dependencies and tools, making it easier to set up your development envrionment and ensuring consistent behavior across different environments.
+
+## Base Docker images
+
+Apify provides several Docker images that can serve as base images for Actors. All images come in two versions:
+
+`latest` - This version represents the stable and productio-ready release of the base image.
+`beta` - This version is intended for testing new features. Use at your own risk.
```

Comment:
This doesn't really render well, perhaps you meant a list here?
<img width="779" alt="Screenshot 2024-05-24 at 9 11 45" src="https://github.com/apify/apify-docs/assets/2934111/e02fe175-8d96-4377-9885-cd2a5681cb40">

```suggestion
- `latest` - This version represents the stable and productio-ready release of the base image.
- `beta` - This version is intended for testing new features. Use at your own risk.
```

Reply:
Thank you, no idea how that slipped by me